client/index.html

```diff
@@ -221,6 +221,12 @@ <h1 class="title">Settings</h1>
 									Show seconds in timestamp
 								</label>
 							</div>
+							<div class="col-sm-7">
+								<label class="opt">
+									<input type="checkbox" name="linkNicks">
+									Make mentioned names clickable
+								</label>
+							</div>
```

Comment:
I know this is gonna add up to my rep, but I'm not at all in favor of a user setting for this.
If this PR works and we want it (I personally do!), let's make some UX decision and simply make this a thing.
Code will be simpler, less branching (i.e. less things to make sure we do not break), and it's a good addition overall. 👍